app/static/src/app/components/code/CodeTab.vue

```diff
@@ -20,16 +21,20 @@
 import { defineComponent, computed } from "vue";
 import { useStore } from "vuex";
 import VueFeather from "vue-feather";
+// eslint-disable-next-line import/no-webpack-loader-syntax
+import codeHelp from "raw-loader!../../help/code.md";
 import CodeEditor from "./CodeEditor.vue";
 import { ModelAction } from "../../store/model/actions";
 import userMessages from "../../userMessages";
 import ErrorInfo from "../ErrorInfo.vue";
 import SelectedVariables from "./SelectedVariables.vue";
 import VerticalCollapse from "../VerticalCollapse.vue";
+import GenericHelp from "../help/GenericHelp.vue";
```

Comment:
I don't love the name GenericHelp, as it's not clear why it's generic - perhaps LongFormHelp? Or DialogHelp or WodinHelp? I don't love any of these alternatives either really...

Reply:
No, I just couldn't think of anything better! I'll put a comment in the component about what 'generic' means and we can rename it later if we think of something better!